packages/react-conformance/src/types.ts

```diff
@@ -5,10 +5,16 @@ import { mount } from 'enzyme';
 
 export type Tests = keyof typeof defaultTests;
 
+export interface TestingOptions {
```

Comment:
One more nitpicky naming thing (sorry), this should probably just be `TestOptions`. And add a doc comment noting that it's options for individual tests.